src/components/UserProfile/BlueSquares/BlueSquare.jsx

```diff
@@ -15,37 +15,48 @@ const BlueSquare = (props) => {
-              <div
-                key={index}
-                role="button"
-                id="wrapper"
-                data-testid="blueSquare"
-                className="blueSquareButton"
-                onClick={() => {
-                  if (!blueSquare._id) {
-                    handleBlueSquare(isInfringementAuthorizer, 'message', 'none');
-                  } else if (canPutUserProfileImportantInfo) {
-                    handleBlueSquare(
-                      canPutUserProfileImportantInfo,
-                      'modBlueSquare',
-                      blueSquare._id,
-                    );
-                  } else {
-                    handleBlueSquare(
-                      !canPutUserProfileImportantInfo,
-                      'viewBlueSquare',
-                      blueSquare._id,
-                    );
-                  }
-                }}
-              >
-                <div className="report" data-testid="report">
-                  <div className="title">{formatDate(blueSquare.date)}</div>
-                  {blueSquare.description !== undefined &&
-                    <div className="summary">{formatDateFromDescriptionString(blueSquare.description)}</div>
-                  }
+
+              .sort((a, b) => (a.date > b.date ? 1 : -1))
+              .map((blueSquare, index) => (
+                <div
+                  key={index}
+                  role="button"
+                  id="wrapper"
+                  data-testid="blueSquare"
+                  className="blueSquareButton"
+                  onClick={() => {
+                    if (!blueSquare._id) {
+                      handleBlueSquare(isInfringementAuthorizer, 'message', 'none');
+                    } else if (canPutUserProfileImportantInfo) {
+                      handleBlueSquare(
+                        canPutUserProfileImportantInfo,
+                        'modBlueSquare',
+                        blueSquare._id,
+                      );
+                    } else {
+                      handleBlueSquare(
+                        !canPutUserProfileImportantInfo,
+                        'viewBlueSquare',
+                        blueSquare._id,
+                      );
+                    }
+                  }}
+                >
+                  <div className="report" data-testid="report">
+                    <div className="title">
+                      
+                      {formatDate(blueSquare.date)}
+                    
+                    
+                    </div>
+                    
+                    {blueSquare.description !== undefined && 
+                      <div className="summary">{blueSquare.createdDate !== undefined ? formatCreatedDate(BlueSquare.createdDate)+":"  : null}
+
+          
+                      {formatDateFromDescriptionString(blueSquare.description)}</div>
+                    }
+                  </div>
                 </div>
               </div>
```

Comment:
I think you may have an extra `</div` closing tag on line 61 that is causing the build to fail.